ITEM.tsv

```diff
@@ -62,7 +62,7 @@ ITEM_20150317_000061	Demion Sword
 ITEM_20150317_000062	Master Sword 
 ITEM_20150317_000063	Panto Sword 
 ITEM_20150317_000064	A crude sword used by the Panto. You could craft a superior version yourself. 
-ITEM_20150317_000065	Flonas Sabre 
+ITEM_20150317_000065	Flona's Sabre 
```

Comment:
https://github.com/Treeofsavior/EnglishTranslation/wiki/2.3.%20Equipment

The original was fine.